_component_design/option-tiles.md

```diff
@@ -0,0 +1,338 @@
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Use these spacing guidelines when you construct option tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Option tile spacing"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/inputs-and-controls/option-tiles/optiontiles-spacing.png" width="604"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Behaviors
+
+Ensure that you construct option tiles so that they conform to the following single- and multi-select behaviors.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Single-select interactions
+
+A single-select interaction is analogous to radios where the user can only select one option tile at a time.
+
+In the default state, the user can click any of the option tiles. You can construct option tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the previously selected option tile becomes deselected and all remaining tiles change to the subdued state. The subdued state reinforces that only one option tile can be selected.
+
+All unselected option tiles change to the hover state as the user scrolls over the option tile.
+
+Saved credit card payment information is a common single-select use case.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Single-select option tile interactions"] [class:"image bg-light border"] %}
+<embed src="{{site.baseurl}}/assets/images/components/inputs-and-controls/option-tiles/optiontiles-behavior-single.png" width="460"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Multi-select interactions
+
+A multi-select interaction is analogous to checkboxes where the user can select multiple option tiles at a time.
+
+In the default state, the user can click any of the option tiles. You can construct option tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the unselected option tiles remain in the default state. The default state reinforces that multiple option tiles can be selected.
```

Comment:
Same here for the bullets/skipping comment above

Reply:
Also shortened this text quite a bit. See how it works for ya!